config/initializers/mailer.rb

```diff
@@ -30,6 +30,9 @@
   # You can use letter opener for your local development by setting the environment variable
   config.action_mailer.delivery_method = :letter_opener if Rails.env.development? && ENV['LETTER_OPENER']
 
+  # Use sendmail if using postfix for email
+  config.action_mailer.delivery_method = :sendmail if ENV['ENABLE_POSTFIX']
```

Comment:
Shall we make this true by default?

Reply:
Rather than introducing the new environment variable, could we do `SMTP_ADDRESS`.blank?

Reply:
@pranavrajs @sojan-official Yeah, that works. The thing is, this will not work out of the box. Port 25 is disabled by all major cloud providers.

Ref: https://deploy-preview-356--chatwoot-handbook.netlify.app/docs/self-hosted/configuration/environment-variables#postfix

Reply:
@vishnu-narayanan That should be okay.